packages/typography/src/Label.tsx

```diff
@@ -73,3 +73,5 @@ export const Label = ({
 };
 
 Label.displayName = 'Label';
+
+export default Label;
```

Comment:
Looks like the `git-cp` command only worked for the `Label` component. Not necessary to fix this, but it'd be nice